src/main/webapp/js/pages/savedPlanners.js

```diff
@@ -56,6 +57,23 @@ function createCard(planName, tableData, creditsData) {
   title.setAttribute('class', 'm-0 font-weight-bold text-primary');
   title.innerText = planName;
   header.appendChild(title);
+  // Create delete button.
+  const deleteButtonElement = document.createElement('button');
+  deleteButtonElement.innerHTML =
+      '<i class="fas fa-trash-alt" id="modeicon"></i>';
+  deleteButtonElement.setAttribute(
+      'class', 'rounded-circle border-0 btn-lg float:right');
```

Comment:
are you sure `float:right` is a class? Should this be a separate CSS style declaration?